content/for-developers/strategies-and-conventions/fonts.mdx

```diff
@@ -1,5 +1,22 @@
 ---
 title: Fonts
-lastUpdatedDate: 2022-04-06T12:00:00.000Z
+lastUpdatedDate: 2022-10-27T12:00:00.000Z
 ---
-Fonts
\ No newline at end of file
+
+
+We have 5 font family classes available that allow one to use Decanter’s font stack.
+
+**Note:** You will still need to import these fonts on your own.
+
+All fonts except the Stanford ligature font are available on [Google Fonts](https://fonts.google.com/). The Stanford font is available through the University Communications CDN.
+
+- **`su-font-sans`**\
+Source Sans Pro, Helvetica Neue, Helvetica, Arial, sans-serif
+- **`su-font-serif`**\
+<span class="su-font-serif">Source Serif Pro, Georgia, Times, Times New Roman, serif</span>
```

Comment:
@jdwjdwjdw @yvonnetangsu What do you think about this – displaying the font stacks with the class? The Stanford one below looks a tad funky but thought I'd offer it up. Thanks!

Reply:
@iamrentman The page is looking good 😄 
Instead of using the font class in the font stack line this way, what do you think about adding an example sentence underneath the font stack, so eg,
```suggestion
<span>Source Serif Pro, Georgia, Times, Times New Roman, serif</span>
<p class="su-font-serif">The quick brown fox jumps over the lazy dog</p>
```
Maybe not exactly that but something similar? And for the Stanford font, we can just display Stanford University (since it doesn't work for any other alphabets). Thanks, Justin!

Reply:
Good idea! I've added a bit to the page. Without the labels – Font stack: and Example: – it looked like a jumbled mess of text so I styled it a bit to help break that up and make it a bit easier to digest. Let me know what you think.